meson.build

```diff
@@ -585,6 +605,16 @@ subdir('src/lxc/tools/include')
 subdir('src/tests')
 subdir('templates')
 
+# Pkg-config.
+pkg_config_file = pkgconfig.generate(liblxc,
+    description: 'linux container tools',
+    version: version_data.get('LXC_VERSION'),
+    url: 'http://linuxcontainers.org',
+    libraries: '-lutil -lpthread -ldl',
+    libraries_private: pkgconfig_libs,
+    install_dir: join_paths(libdir, 'pkgconfig')
```

Comment:
pkgconfig.generate() defaults to `join_paths(get_option('prefix'), get_option('libdir'), 'pkgconfig')` already (unless liblxc has its own custom install_dir), so this kwarg is redundant.